DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/Index.cshtml

```diff
@@ -8,7 +8,7 @@
   ViewData["Application"] = "Tracking System";
   ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
   ViewData["HeaderPathName"] = "Tracking System";
-  var hasSixPrompts = Model.CustomFields.Count == 6;
+  var canAddNewPrompt = Model.CustomFields.Count == 6;

```

Comment:
Sorry, my renaming suggestion also needs the logic negated 🤦 .

Reply:
Should be strictly less than 6 right?

Reply:
I should have never renamed it 😢